boards/shields/m5stack_core2_ext/m5stack_core2_ext.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&m5stack_mbus_i2c0 {
+	status = "okay";
+
+	mbus_mpu6886: mbus_mpu6886@68 {
```

Comment:
Sorry, I missed it, this must be in the form `<device>_<shield>`, ` mpu6886_m5stack_core2_ext: mpu6886@68 {`

Reply:
Sorry for the late response. I was on vacation.
PR has been updated